zgw_consumers/api_models/zaken.py

```diff
@@ -94,3 +94,14 @@ def get_betrokkene_type_display(self):
 
     def get_omschrijving_generiek_display(self):
         return RolOmschrijving.values[self.omschrijving_generiek]
+
+
+@dataclass
+class ZaakInformatieObject(ZGWModel):
+    url: str
+    informatieobject: str
+    zaak: str
+    aard_relatie_weergave: str
+    titel: str
+    beschrijving: str
+    registratiedatum: str
```

Comment:
```suggestion
    registratiedatum: datetime
```
the factories take care of the type mapping/conversion, see https://github.com/maykinmedia/zgw-consumers/blob/master/zgw_consumers/api_models/base.py#L26